src/utils/Logo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function Logo(props) {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3" width="841.9" height="595.3" {...props}>
+      <g fill="#61DAFB">
```

Comment:
Dude, why don't you just use `import logo.svg`

Reply:
removed